docs/application/dotnet/guides/nui/nui-components/Scrollbar.md

```diff
@@ -1,146 +0,0 @@
-# ScrollBar

```

Comment:
Would you please check the file again?

According to the ACR, the scrollbar under Native is to be removed.

However the scrollbar of .NET was removed.

Reply:
> Would you please check the file again?
> 
> According to the ACR, the scrollbar under Native is to be removed.
> 
> However the scrollbar of .NET was removed.

I'm Sorry. I can't understand "According to the ACR, the scrollbar under Native is to be removed."
In ACR-332, src\Tizen.NUI.Components\Controls\ImageScrollBar.cs deprecated, it's ScrollBar of .NET

Reply:
I went to ACR-332 instead of TCSACR-332, and only checked that the link is for Native.

Now I see https://code.sec.samsung.net/jira/browse/TCSACR-332 is for this PR and updated the title of the issue.
Thanks for checking.